src/pykeen/models/base.py

```diff
@@ -1202,3 +1202,15 @@ def _can_slice(fn) -> bool:
 
 class MultimodalModel(Model, autoreset=False):
     """A multimodal KGE model."""
+
+    def score_hrt(self, hrt_batch: torch.LongTensor) -> torch.FloatTensor:  # noqa: D102
```

Comment:
this code was duplicated on both distmultliteral and complexliteral so might as well deduplicate via the base class